include/deal.II/lac/petsc_sparse_matrix.h

```diff
@@ -27,6 +27,9 @@
 
 #    include <vector>
 
+#    include "petscis.h"
+#    include "petscistypes.h"
```

Comment:
I'm surprised clang-format accepted this - its probably because of the quotes. Could you switch to using brackets with these inclusions? That should fix it.

Reply:
done!